core/payment-driver/zksync/src/driver.rs

```diff
@@ -128,8 +128,9 @@ impl PaymentDriver for ZksyncDriver {
         _caller: String,
         msg: Exit,
     ) -> Result<String, GenericError> {
-        log::info!("EXIT = Not Implemented: {:?}", msg);
-        Ok("NOT_IMPLEMENTED".to_string())
+        wallet::exit(_caller, &msg)
+            .await
+            .and(Ok("Success!".to_string()))
```

Comment:
Or should I return link to [[zkscan](https://rinkeby.zkscan.io/)] ...

Reply:
I believe `wallet::exit` should return transaction hash and here it should be formatted into some kind of nice message like "Withdrawal has been accepted by the zkSync operator. It may take some time until the funds are available on Ethereum blockchain. Tracking link: {}"